openff/toolkit/tests/test_toolkit_io.py

```diff
@@ -26,7 +26,11 @@
 from openff.toolkit.tests import create_molecules
 from openff.toolkit.tests.utils import requires_openeye, requires_rdkit
 from openff.toolkit.topology.molecule import Molecule
-from openff.toolkit.utils import OpenEyeToolkitWrapper, RDKitToolkitWrapper, exceptions
+from openff.toolkit.utils import OpenEyeToolkitWrapper, RDKitToolkitWrapper
+from openff.toolkit.utils.exceptions import (
```

Comment:
Preference, not performance, I think; I tend to prefer importing only things that are needed in a particular scope. This module should be pretty lightweight at import time so it should be a wash in terms of performance. Also, even though this can go either way, I have a marginal preference for keeping things consistent without a project and for whatever reason everywhere else uses this pattern. Then again, importing each exception one-by-one was the only way to import them before everything was collected into one namespace ...

Reply:
Fair enough, consistency is a good design pattern!